### PR TITLE
doc: west v0.11.1 notes

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
This is a stopgap release meant to backport bug fixes and new features while v0.12.0 is blocked.

### CODE REVIEW

**Review by cfriedt — APPROVED** [review]

File: .github/workflows/doc-build.yml
Comment (mbolivar-nordic) [inline .github/workflows/doc-build.yml:29]: It's fine to merge with this as-is. There are no doc-build visible changes from a1 to the final release. I'd like to get this merged without having to wait for CI again and will update it tomorrow after the docs include the 0.11 notes, so I can send the release email to the users.

Comment (cfriedt) [reply .github/workflows/doc-build.yml:29]: I don't have permissions to merge manually yet. @galak - can we get another review so I can merge?

**Review by gmarull — APPROVED** [review]

**Pull request merged**
